notebooks/02-transformers.ipynb

```diff
@@ -89,13 +91,12 @@
    },
    "outputs": [],
    "source": [
-    "!pip install transformers\n",
-    "!pip install --upgrade tensorflow"
+    "# !pip install transformers"
```

Comment:
I think it's worth it to pin transformers to the current version since we don't test the notebooks -> !pip install transformers==4.0.0 . Wdyt?

Reply:
The goal is still to have those work on the latest release. I'm afraid the pinning will be forever left there afterward and the notebook will never be updated.

Reply:
hmm, yeah, can we add a test to make sure the notebooks always work with the latest version? I'm a bit worried that we'll just forget to update the notebook and it doesn't work which will churn away many users

Reply:
The problem is the notebooks all have a big training, which takes too long to run. It's easy to have something that execute a given notebook, but if we have to skip some cells because of the time it takes, we usually can't execute the cells after.

So we can automate testing on those that take not too long (might require a machine with a GPU) but not the bigger ones.